frontend/src/metabase/ui/components/inputs/MultiAutocomplete/util.ts

```diff
@@ -0,0 +1,17 @@
+/* eslint-disable-next-line import/no-unresolved */
```

Comment:
Eslint is not picking up on the `csv-parse/sync` package for some reason, even though it has a valid `exports` field in it's `package.json`